shared/oae/bundles/ui/default.properties

```diff
@@ -257,6 +257,8 @@ ADD = Add
 ADD_TO_FOLDER = Add to folder
 ADMINISTRATION_PANEL = Administration Panel
 ALIAS_COLON = Alias:
+ALREADY_AT_AN_INSTITUTION = Already at an institution?
+ALREADY_AT_AN_INSTITUTION_INSTRUCTION = Choose your instution and sign in with the same details you use there.
```

Comment:
I'm tempted to change this to : `Choose your institution and sign in with your institutional account`. Not all tenants will allow signing in with their institutional account, but it's as accurate as the current one.